### DISCUSSION TITLE
Adding latest model build/run-time of a node in the lineage UI

### DISCUSSION THREAD

**Discussion by lassebenni** [AltimateAI/vscode-dbt-power-user#1522] [Ideas]
Hi all, 

love the extension first of all, it's already invaluable to me.

What I wondered is how much work it would be to add a detail to the nodes in the lineage UI displaying when the node (model) was last successfully built. That way you can more easily see if you need to rebuild a model or can potentially skip it when you are building them individually to save time and compute.

![image](https://github.com/user-attachments/assets/bf84f2f7-686c-461c-a690-b23c74709ef4)

e.g. if I want to rebuild the upstrem models from `competition_federation_team`, such as `dim_club` as in the picture above, I currently cannot see if that is necessary since I don't know when `dim_club` was successfully built the last time. If I knew that it was built 5 minutes ago as an upstream for another model I ran (and with a large amount of upstream you quickly lose the full picture), I would not need to rerun it, and save a lot of time instead of simply rerunning all upstream models again:

![image](https://github.com/user-attachments/assets/765cbae1-328b-4998-89f6-48d2db85581f)

In the image above I can see that the `dim_club` model was run on 2024-12-11 at 08:40 AM. So I know that I can skip that if I am building the models individually and only need to build `dim_competition` and its upstream.

I don't know how complex this would be. But I think it would be very valuable.

Kind regards

**Comment by anandgupta42** [comment]
@lassebenni  This is a good idea. I will discuss it with the team.
